src/ocean/io/compress/Lzo.d

```diff
@@ -243,6 +243,23 @@ class Lzo
     }
 }
 
+/// Simple compress / decompress example.
+unittest
+{
+    auto lzo = new Lzo;
+    auto original = "compress this!".dup;
```

Comment:
The `.dup` should not be needed.

Reply:
Right, true, it doesn't need to be mutable.

Reply:
Updated.